src/index.ts

```diff
@@ -22,18 +22,33 @@ router.post('/api/player', async function addPlayer (ctx, next) {
     await db.write(player_name, fields)
     ctx.status = 200
     ctx.body =  {update: "update successful"}
+    ctx.response.redirect("/add")
 });
-router.get('/', async function home (ctx, next) {
+router.get('/add', async function add (ctx, next) {
     ctx.body = `
-    <h1>hello world</h1>
+    <h1>Add a player here</h1>
     <form method="POST" action="/api/player">
-    Player Name: <input type="text" name="name"/> <br/>
-    Speed: <input type="text" name="speed"/> <br/>
-    On Base: <input type="text" name="onBase"/> <br/>
-    <button type="submit"> Submit </button>
+        Player Name: <input type="text" name="name"/> <br/>
+        Speed: <input type="text" name="speed"/> <br/>
+        On Base: <input type="text" name="onBase"/> <br/>
```

Comment:
So is type checkbox, but u may not need that?

Reply:
confusing name, but it's a numerical value from 5-13

Reply:
mm. you can also add min=5 max=13 to constrain. `mdn.io` is _awesome_.  i often time guess the URI for information i need and actually get it right.  in this case, mdn.io/input has all sorts of goodies.  whats nice is that if the user has violated your contraints, the form wont submit. it will guide their eyes to their violation too